VK/VK/FriendsList/Model/UserInfo.swift

```diff
@@ -1,11 +1,12 @@
-// FriendsInfo.swift
+// UserInfo.swift
 // Copyright © RoadMap. All rights reserved.
 
 import Foundation
 
 /// Information about friend
-struct FriendsInfo {
+struct UserInfo {
     var name: String
     var avatarName: String
     var likesCount: Int
+    var imagesNames: [String] = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
```

Comment:
```suggestion
    var imagesNames = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
```
Явную аннотацию типа лучше не указывать когда и так тип понятен